envoy_reader/envoy_reader.py

```diff
@@ -356,11 +358,10 @@ async def inverters_production(self):
                 self.password = self.serial_number_last_six
 
         try:
-            response = requests_sync.get(
```

Comment:
Alternative fix is to run `requests_sync.get` in an asyncio Executor with [`asyncio.run_in_executor()`

Reply:
@balloob Is that a better solution?  It seems my changes work, but I'm not very familiar with the intricacies of threading and python.

Reply:
It's better because you need one dependency less 🙂 it's the official way for running threaded code from asyncio

Reply:
@balloob Okay, sounds good.  I can look into it in a week or so, been busy with work and school lately